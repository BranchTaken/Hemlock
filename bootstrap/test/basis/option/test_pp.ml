open! Basis.Rudiments
open! Basis
open Option
open Format

let test () =
  printf "@[<h>";
  printf "Some 42 -> %a\n" (xpp Uns.xpp) (Some 42L);
  printf "None -> %a\n" (xpp Uns.xpp) None;
  printf "@]"

let _ = test ()
