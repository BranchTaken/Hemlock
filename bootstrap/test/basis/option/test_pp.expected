Some 42 -> Some (42)
None -> None
